upi/openstack/bootstrap.yaml

```diff
@@ -32,13 +32,6 @@
       flavor: "{{ os_flavor_master }}"
       userdata: "{{ lookup('file', os_bootstrap_ignition) | string }}"
       auto_ip: no
+      floating_ips: "[{{ os_bootstrap_fip | default(omit) }}]"
```

Comment:
What happens if we set `os_bootstrap_fip` without setting `os_external_network`? Same question for `os_api_fip` and `os_ingress_fip`.
Should we add a playbook to validate the configuration is correct?

Reply:
We state in the docs that it's required (thanks to your comment above). I don't think we should perform all the checks, as the more complex the playbooks become, the harder it becomes to edit them. WDYT?